pkg/preflight/utils.go

```diff
@@ -14,6 +14,16 @@ func IsGKE() bool {
 	return Instance().ProviderName() == string(cloudops.GCE) && Instance().K8sDistributionName() == gkeDistribution
 }
 
+// IsGKE returns whether the cloud environment is running PKS
+func IsPKS() bool {
+	return Instance().K8sDistributionName() == pksDistribution
+}
+
+// IsAzure returns whether the cloud environment is running on Azure
+func IsAzure() bool {
+	return Instance().ProviderName() == string(cloudops.Azure)
+}
```

Comment:
Do we need azure code for this PR?

Reply:
Added that so we can check for Azure and if we are running a PX image of 3.1.0 for pre-flight.   Using this so I don't have include cloudops in storagecluster.